src/libs/events.js

```diff
@@ -76,7 +76,9 @@ const eventsList = {
   permissionsSynchronizationDelayDisplayed: 'permissions:propagationDelay',
   resourceLeave: 'resource:leave',
   userLogin: 'user:login',
+  userLogout: 'user:logout',
   userRegister: 'user:register',
+  userTimeoutLogout: 'user:timeoutLogout',
```

Comment:
nit - I don't know why I care about this, but my preference would be to just call it `user:timeout`

Reply:
I don't really like the name I used, but I thought there was value in being explicit with the event name since I thought that `user:timeout` was too generic and may not describe the event we want to track clearly enough.

Reply:
Minor comment -- it would be cool to add some instrumentation around `auth.reloadAuthToken()` as well, since failures there may be contributing to the double-login bug (see my comment in [JIRA](https://broadworkbench.atlassian.net/browse/ID-557)). Maybe `user:reload`?